readme.md

```diff
@@ -941,6 +941,7 @@
 - [WebXR](https://github.com/msub2/awesome-webxr#readme) - Enables immersive virtual reality and augmented reality content on the web.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Documentation projects built and published on Read the Docs, topics include both science and software.
```

Comment:
An alternative that might give more context:

```suggestion
- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Documentation projects for software, science and academia, built and published on Read the Docs Community and with accompanying source repositories containing their contents and configuration.
```

Reply:
Any reviewers that would have a helpful opinion here?

Reply:
I think it is a bit long. It could be shorter like you have in your header `Example of Read the Docs for learning and bootstrapping documentation projects.`

Reply:
You are right, it's too long :+1: 

```suggestion
- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects for inspiration, learning and bootstrapping new documentation projects on Read the Docs.
```